packages/tempus/src/Tempus_StepperDIRK_decl.hpp

```diff
@@ -79,6 +79,10 @@ namespace Tempus {
  *   - end for
  *   - \f$x_n \leftarrow x_{n-1} + \Delta t\,\sum_{i=1}^{s}b_i\,\dot{X}_i\f$
  *   - Solve \f$f(\dot{x}_n,x_n,t_n)=0\f$ for \f$\dot{x}_n\f$ [Optional]
+ *
```

Comment:
I'm not sure I understand this comment for DIRK stepper

Reply:
In many of the stepper algorithms, I mention the possibility of updating xDot at the end of the step, though in most instances we do not have that code implemented.  I have been debating on removing these notes until that code is implemented and working.  I think I will remove them to reduce confusion.